src/ploomber/__init__.py

```diff
@@ -16,16 +16,8 @@
 logging.getLogger(__name__).addHandler(logging.NullHandler())
 
 __all__ = [
-    'DAG',
-    'Env',
-    'SourceLoader',
-    'load_env',
-    'with_env',
-    'DAGConfigurator',
-    'InMemoryDAG',
-    'OnlineDAG',
-    'OnlineModel',
-    'lazily_load_entry_point',
+    'DAG', 'Env', 'SourceLoader', 'load_env', 'with_env', 'DAGConfigurator',
+    'InMemoryDAG', 'OnlineDAG', 'OnlineModel', 'lazily_load_entry_point'
```

Comment:
the trailing comma is on purpose, it makes yapf to format one element per line, which makes it more readable

Reply:
Weird, this how the auto yapf watcher formatted it. I'll change to the old format.